.github/workflows/ci-cd.yml

```diff
@@ -26,6 +26,19 @@ env:
 
 permissions: {}
 jobs:
+  backport:
+    runs-on: ubuntu-latest
+    name: Backport label added?
+    if: ${{ github.base_ref == 'master' }}
+    steps:
+    - name: Check labels
+      run: |
+        for label in "${{ github.event.pull_request.labels }}"; do
+            case $label in backport*)
+                exit 0
+            esac
+        done
+        exit 1
```

Comment:
@webknjaz Let me know if you've got any ideas how to do this correctly, I'm just guessing.